Packs/ForescoutEyeInspect/IncidentFields/incidentfield-ForescoutEyeInspect_Status.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "incident_forescoutstatus",
```

Comment:
@RazOmry We are trying to minimize the creation of new fields. Please use the common field "Source Status" instead.